ci/travis/before_install-osx.sh

```diff
@@ -42,6 +42,6 @@ curl --silent --show-error --retry 5 -O http://releases.numenta.org/pip/1ebd3cb7
 python get-pip.py --user pip setuptools wheel
 
 echo "Installing Python dependencies"
-pip install --use-wheel --user -r bindings/py/requirements.txt --quiet || exit
+pip install --user -r bindings/py/requirements.txt --quiet || exit
```

Comment:
Why the change in travis ci config?

Reply:
This was part of the `pip 10` fix and replaced everywhere this option was used.
This file is current part of the codebase and if we restore travis functionality this command will fail. 
If we don't want to support travis then i recommend deleting all travis related files to avoid confusion